ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -1299,13 +1324,18 @@ object JdbcLedgerDao {
       dbType,
       loggerFactory)
 
+  private val PARTY_SEPARATOR = '%'
```

Comment:
What are the chances that we'll decide to relax DAML party identifier restrictions so that `%` is a valid character, while forgetting about his line?

Reply:
Non-zero 😈